transport/response.hh

```diff
@@ -95,9 +93,9 @@ private:
     void compress_snappy();
 
     template <typename CqlFrameHeaderType>
-    sstring make_frame_one(uint8_t version, size_t length) {
-        sstring frame_buf = uninitialized_string(sizeof(CqlFrameHeaderType));
-        auto* frame = reinterpret_cast<CqlFrameHeaderType*>(frame_buf.data());
+    temporary_buffer<char> make_frame_one(uint8_t version, size_t length) {
+        temporary_buffer<char> frame_buf(sizeof(CqlFrameHeaderType));
```

Comment:
This adds an allocation, but it cancels out with the one removed in scattered_message::append().